<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interpreters</class>
 <widget class="QWidget" name="Interpreters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ëß£ÈáäÂô®</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="py_table">
       <column>
        <property name="text">
         <string>ÂêçÁß∞</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ë∑ØÂæÑ</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="add_btn">
         <property name="text">
          <string>‚ûï</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="del_btn">
         <property name="text">
          <string>‚ûñ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="change_btn">
         <property name="text">
          <string>üñä</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
